manifests/params.pp

```diff
@@ -23,7 +23,13 @@
       $sentinel_daemonize        = true
       $sentinel_init_script      = '/etc/init.d/redis-sentinel'
       $sentinel_package_name     = 'redis-sentinel'
+
+      if versioncmp($facts['os']['release']['full'], '10.0.0') >= 0 {
+      $sentinel_pid_file         = '/var/run/sentinel/redis-sentinel.pid'
```

Comment:
```suggestion
      $sentinel_pid_file         = '/run/sentinel/redis-sentinel.pid'
```

Reply:
This was marked as resolved, but I don't see the actual change so I've marked it as unresolved. Mind having a look?